R/rdeephaven/src/client.cpp

```diff
@@ -208,8 +481,20 @@ class ClientWrapper {
         return new TableHandleWrapper(new_tbl_hdl);
     }
 
+    /**
+    * Shuts down the Client and all associated state (GRPC connections, subscriptions, etc).
+    * This method is used if a caller wants to shut down Client state early. If it is not called,
+    * the shutdown actions will happen when this Client is destructed. The caller must not use any
+    * associated data structures (TableHandleManager, TableHandle, etc) after close() is called or
+    * after Client's destructor is invoked. If the caller tries to do so, the behavior is
+    * unspecified.
+    */
+    void close() {
+        internal_client.close();
+    }
+
 private:
-    const deephaven::client::Client internal_client;
+    deephaven::client::Client internal_client;
```

Comment:
Because of this:
```
client.cpp: In member function ‘void ClientWrapper::close()’:
client.cpp:454:31: error: passing ‘const deephaven::client::Client’ as ‘this’ argument discards qualifiers [-fpermissive]
  454 |         internal_client.close();
      |                               ^
In file included from client.cpp:13:
/home/alexpeters/deephaven/dhcpp/local/deephaven/include/deephaven/client/client.h:208:8: note:   in call to ‘void deephaven::client::Client::close()’
  208 |   void close();
      |        ^~~~~
```

Probably I just don't know C++.

Reply:
Looks like other methods in libs would need const.  It is a good feature, but it frequently does not get used.

I this case, it might be correct to not have const, since Close does change the object.